api/v1/hardening_types.go

```diff
@@ -41,5 +41,9 @@ type CBContainersEnforcerSpec struct {
 	// +kubebuilder:default:=<>
 	Probes CBContainersHTTPProbesSpec `json:"probes,omitempty"`
 	// +kubebuilder:default:=5
-	WebhookTimeoutSeconds int32 `json:"webhookTimeoutSeconds,omitempty"`
+	WebhookTimeoutSeconds    int32 `json:"webhookTimeoutSeconds,omitempty"`
+	EnableEnforcementFeature bool  `json:"enableEnforcementFeature,omitempty"`
```

Comment:
I know that is false by default, by maybe it good to be explicit here?
wdyt?

Reply:
Forgot to add a comment. We discussed it but adding `+kubeuilder:default` means the field is always populated in the spec (even if not specified) - so creating a CR will have this field on subsequent `get cbcontaineragents <x> -o yaml> `calls

Since this is an opt-in feature (and not even GA yet), I thought it's better to not have in the CR yet unless explicitly specified